interfaces/IDeviceInfo.h

```diff
@@ -10,7 +10,7 @@ namespace Exchange {
     struct EXTERNAL IDeviceCapabilities : virtual public Core::IUnknown {
         enum { ID = ID_DEVICE_CAPABILITIES };
 
-        virtual ~IDeviceCapabilities() {}
+        virtual ~IDeviceCapabilities() override = default;
```

Comment:
Hi Wissem, really small thing but if we add override we remove the virtual in front :) I know it is allowed but it is kinda double then, that is why we remove it (and sorry for nagging but better to keep the code consistent, certainly if people start copying code)